customfiles/14_0_INSTALLERCONFIG.sample

```diff
@@ -160,6 +165,40 @@ rcvar=firstboot_depenguin_enable
 start_cmd="firstboot_depenguin_run | logger -s -t 'depenguin'"
 stop_cmd=":"
 
+firstboot_depenguin_is_realtek_ifname()
+{
+	expr "X$1" : '^Xre[0-9]' >/dev/null
+}
+
+firstboot_depenguin_check_realtek_pci_id()
+{
+	local if_re_count
+
+	if_re_count==$(/usr/sbin/pciconf -l | \
```

Comment:
double equals

Reply:
Sad typo when copying from the code I tested with, sorry